script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -37,7 +37,7 @@ Having a girl around my own age look after me isn't to my taste right now.
 Hisui bows deeply.
 The stabbing pain in my chest is caused by my selfish guilt.
 <Page11>
-"All right. From the way Akiha puts it, she's not going to let me be alone, and I don't think she'll let me get rid of you either. So I'll accept your help but―――"
+"All right. From the way Akiha puts it, she's not going to let me be alone. In the first place, I don't think making her dismiss you because I feel inconvenienced is a good thing. So I'll accept your help but―――"//「分かった。秋葉のあの言いぶりじゃ俺を一人にはしないだろうし、そもそも俺の気持ちの問題で君を解雇させるのは良くはないと思う。ありがたくお世話になるけど―――」
```

Comment:
In the first place sounds a bit JTE, but then it's hard to directly translate words like that and avoid it. Also, is inconvenience correct here? Should we not emphasize that it's just his feelings that are the problem?

> Plus, I don't think making her dismiss you because I feel inconvenienced[?] is a good thing [after all].

Reply:
Yeah, "In the first place" isn't great. I like "plus" a lot more.
What would you propose for "俺の気持ちの問題で", I got stuck trying to find a good formulation for it, that's why I settled for "inconvenienced".

Reply:
Uncomfortable? Can't think of a great turn of phrase. Inconvenient seems off because having a maid is technically _convenient_, he just doesn't like it. Maybe one of the other reviewers can come up with something nicer

Reply:
>Plus, I don't think making her dismiss you just to take my feelings into consideration is a good thing.

>Plus, I don't think making her dismiss you just to make allowance for me is a good thing.